packages/context-editor/src/ContextEditor.tsx

```diff
@@ -155,15 +170,19 @@ function UnwrappedEditor(props: ContextEditorProps) {
 
 	useEffect(() => {
 		/* Every Render */
-		if (view.current) {
-			view.current.setProps({
-				editable: () => !props.disabled,
-			});
-			const tr = view.current.state.tr
-				.setMeta(reactPropsKey, { ...props, suggestData, setSuggestData })
-				.setMeta(attributePanelKey, { panelPosition, setPanelPosition });
-			view.current?.dispatch(tr);
-		}
+
+		setTimeout(() => {
```

Comment:
adding this fixes a `flushSync` error. i don't really know why, and avoiding that kind of nonsense does seem like a strong pro of using the other view library